autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -90,9 +90,10 @@ def get_preset_models_classification(path, problem_type, objective_func, stoppin
                                   objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=nn_options.copy()),
         )
     if lr_options is not None:
-        models.append(
-            LinearModel(path=path, name='LinearModel', problem_type=problem_type, objective_func=objective_func, hyperparameters=lr_options.copy())
-        )
+        for lr_option in lr_options:
```

Comment:
We have to ensure models don't share the same name

Reply:
Addressed in the last push